x509/x509_test.go

```diff
@@ -79,6 +80,34 @@ func TestParsePKIXPublicKey(t *testing.T) {
 	}
 }
 
+func TestParsePKIXPublicKeyEd25519(t *testing.T) {
+	block, _ := pem.Decode([]byte(pemPublicKeyEd25519))
+	pub, err := ParsePKIXPublicKey(block.Bytes)
+	if err != nil {
+		t.Errorf("Failed to parse Ed25519 public key: %s", err)
+		return
```

Comment:
Observation: I guess `t.Fatalf()` wasn't a thing when this file was written?

Reply:
Could be!  Anyway, fixed (and converted errors to canonical form).